ui/css/design-system/colors.scss

```diff
@@ -97,6 +97,7 @@ $success-2: #caf4d1;
 $success-3: #219e37;
 
 $ui-black: #24292e;
+$ui-grey: #d6d9dc;
```

Comment:
Went with the value for `$Grey-100` here, for consistency with `$ui-black` (which equals `$Black-100`)